AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginFlutterIntegrationTests/TestSupport/AmplifyFlutterHelpers/FlutterSerializedModel.swift

```diff
@@ -0,0 +1,231 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+import Foundation
+import Amplify
+
+/// `FlutterSerializedModel` assists in serializing and deserializing JSON when interacting with the amplify-ios DataStore API.
+/// Taken from [amplify-flutter DataStore category](https://github.com/aws-amplify/amplify-flutter/blob/main/packages/amplify_datastore/ios/Classes/types/model/FlutterSerializedModel.swift)
+struct FlutterSerializedModel: Model, JSONValueHolder {
```

Comment:
Will this ever change in Flutter? If so, have you thought about how we should maintain this class in both places? ie. Do we need a comment in the Flutter codebase to make sure if modified, we should change and run integration test in iOS?

Reply:
@lawmicha I don't think the validity of these tests would be impacted by a change to FlutterSerializedModel, but I still think your suggestion for [a comment in the flutter codebase](https://github.com/aws-amplify/amplify-flutter/pull/1192) is a good one.